dhall/dhall.cabal

```diff
@@ -510,6 +511,8 @@ Library
         Dhall.Import.Types,
         Dhall.Eval,
         Paths_dhall
+    Autogen-Modules:
+        Paths_dhall
```

Comment:
@vmchale Would you mind explaining why you added this Autogen-Modules section? This might be relevant for my patch for #1331.